chemprop/cli/common.py

```diff
@@ -9,39 +10,40 @@
 
 
 def add_common_args(parser: ArgumentParser) -> ArgumentParser:
-    data_args = parser.add_argument_group("input data parsing args")
+    data_args = parser.add_argument_group("shared input data args")
     data_args.add_argument(
         "-s",
         "--smiles-columns",
         nargs="+",
-        help="The columns in the input CSV containing SMILES strings. If unspecified, uses the the 0th column.",
+        help="The column names in the input CSV containing SMILES strings. If unspecified, uses the the 0th column.",
     )
     data_args.add_argument(
         "-r",
         "--reaction-columns",
         nargs="+",
-        help="The columns in the input CSV containing reactions.",
+        help="The column names in the input CSV containing reaction SMILES in the format 'REACTANT>AGENT>PRODUCT'",
```

Comment:
Should we add that the reaction SMILES is 'REACTANT>>PRODUCT' is no AGENT, or is that implied?

Reply:
As I understand, they could include an AGENT. This [line](https://github.com/chemprop/chemprop/blob/34872d4a04473d75a51ffbd2385e24db410dd191/chemprop/data/datapoints.py#L123) joins the agent smiles to the reactant smiles (with a ".") if the AGENT is specified. 
Or maybe your question was should we add that you don't need the AGENT. I think if users know what a reaction SMILES, they probably know that the AGENT part of the SMILES is optional.

Reply:
I decided to change it to more explicit.